### PR TITLE
Rename state to zipCode to avoid confusion

### PR DESCRIPTION
**Pull request by vasanthk**
Pretty minor. But destructuring state using the regex was a little bit ambiguous since there is no state in the address (zip code instead).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,16 +102,17 @@ setTimeout(() => {
 **Bad:**
 ```javascript
 const address = 'One Infinite Loop, Cupertino 95014';
-const cityStateRegex = /^[^,\\]+[,\\\s]+(.+?)\s*(\d{5})?$/;
+const city
+Regex = /^[^,\\]+[,\\\s]+(.+?)\s*(\d{5})?$/;
```
Comment (vsemozhetbyt) [inline README.md]: It seems to be erroneous line break here

**Comment by vasanthk** [comment]
@vsemozhetbyt My bad. Fixed it.

File: README.md [hunk 2]
```diff
@@ -102,8 +102,7 @@ setTimeout(() => {
 **Bad:**
 ```javascript
 const address = 'One Infinite Loop, Cupertino 95014';
-const city
-Regex = /^[^,\\]+[,\\\s]+(.+?)\s*(\d{5})?$/;
+const cityStateRegex = /^[^,\\]+[,\\\s]+(.+?)\s*(\d{5})?$/;
```
Comment (ryanmcdermott) [inline README.md]: The name of the Bad regex and Good regex don't match. We should make them the same variable.

**Comment by vasanthk** [comment]
@ryanmcdermott Done!

**Comment by ryanmcdermott** [comment]
Woo hoo! Thank you Vasa!

**Pull request merged**